client/src/components/HandMarkedPaperBallot.tsx

```diff
@@ -442,6 +444,14 @@ export const CandidateContestChoices = ({
   )
 }
 
+function hasVote(vote: Vote | undefined, optionId: string): boolean {
+  return (
+    vote?.some((choice: Candidate | string) =>
+      typeof choice === 'string' ? choice === optionId : choice.id === optionId
+    ) ?? false
+  )
+}
+
```

Comment:
Ideally, should this code be placed before it's used?

Reply:
You can if you like. Function declarations (but not `const` function variables) are hoisted to the top of their scope so you can use them even "before" they are defined.

Reply:
Thanks. Yep. Don't have a strong preference, I've seen linters complain before.